### ISSUE TITLE
Compare runv with Clear Container(Pod support)

### ISSUE THREAD

**Issue by jovizhangwei** [hyperhq/runv#565]
Hi,

I'm a newbie for runv, may I ask the differences between runv with Clear Container? like does it support Pod in k8s? etc. I did't found any doc about Pod support or plans in runv.

Thanks.

**Comment by gnawux** [comment]
@jovizhangwei I think you may ref the following projects:

- kubernetes/frakti: https://github.com/kubernetes/frakti : run kubernetes with hypercontainer (runV)
- openstack/stackube: https://github.com/openstack/stackube: a kubernetes centric openstack distro, support hard multi-tenancy with hypercontainer(runV) and overlay network
- hyperhq/hypernetes: https://github.com/hyperhq/hypernetes: a multi-tenant kubernetes distro based on hypercontainer(runV)

Compare with clear container:

- both projects aims to implement OCI spec with hypervisor, and we collaborate each other and share some common infrastructure, such as hyperhq/hyperstart in guest;
- both projects are open sourced and both of them were announced in May 2015;

For runV:

- we support full kubernetes CRI semantics as some of the kubernetes CRI developers are hyperd and runV developers from hyperhq as well.
- we are vendor neutral and support different hypervisors and different CPU architectures.
- and some implementation detail differences.

**Comment by jovizhangwei** [comment]
Thanks @gnawux for give me helpful answers.

I noticed that Clear Container still have many limitations, so just wondering whether runv have the same issues here, for example, support "docker run -m", "docker run --cpus", and "docker stats".

I tried to found these info in runv's Doc, but got nothing about these sample usages, it would be very useful to show more docker CLI usage in README.md or in Doc.

Thanks.
